adapter/parser/src/main/java/org/mobilitydata/gtfsvalidator/parser/GtfsEntityParser.java

```diff
@@ -185,6 +189,13 @@ public ParsedEntity parse(RawEntity toParse) {
                                 ));
                     }
 
+                } else if (columnSpecProto.getType().getType() ==
```

Comment:
My understanding of the logic is that line 200 `contentByHeaderMap.put(columnSpecProto.getName(), rawField);` was supposed to let through any color raw value and that invalid color would be caught by https://github.com/MobilityData/gtfs-validator/blob/e2e59464ea7955feffa5af8ed5a2b48fc95a5104/adapter/validator/src/main/java/org/mobilitydata/gtfsvalidator/validator/GtfsTypeValidator.java#L136
Here it seems we would discard invalid raw values without generating an output